bin/controller/unittest/Makefile

```diff
@@ -14,7 +14,7 @@ USE_RAPIDJSON=1
 
 all: build
 
-LIBS=\
+LIB_OBJS=\
```

Comment:
I find `LIB_OBJS` quite a confusing name.  I can see it came from `LIBS`.  But adding `OBJS` on the end makes it sound like these objects come from our `libs`, which they don't.

I think it would be better to use `EXT_OBJS` (ext short for external) or `TESTED_OBJS` (since they're the objects being tested) or `PROG_OBJS` (since we only use this pattern for the unit tests of programs, not libraries).

Of these I actually think `TESTED_OBJS` will be clearest to somebody reading `stdboosttest.mk` in the future without referring to this PR.

Reply:
👍  `TESTED_OBJS` it is